### DISCUSSION TITLE
Why  i have no 'chatgpt' in translator option?

### DISCUSSION THREAD

**Discussion by femto354** [dmMaze/BallonsTranslator#270] [Q&A]
how to add chatgpt translator?
![scgithub](https://github.com/dmMaze/BallonsTranslator/assets/149766904/49bf7a6f-98ef-4864-8c70-eacfa937a14b)

**Comment by dmMaze** [comment]
Please read [README](https://github.com/dmMaze/BallonsTranslator/blob/dev/README.md) (recently updated) to install the latest version

**Reply by femto354** [reply]
okay, thx
i love you so much
